backend/bot/index.js

```diff
@@ -36,12 +68,61 @@ const postResultToSlack = (match) => {
     prefix = 'HOLY SHIT! ';
   }
 
-  const suffix = isComedyDuo?  ':marioluigi:' : '';
+  const suffix = isComedyDuo ? ':marioluigi:' : '';
 
   const messageText = `${prefix}${winningPlayers.join(', ')} just beat ${losingPlayers.join(', ')}. Rating change: ${ratingChange}${suffix}`
-  bot.postMessage(channel.id, messageText, {as_user: true});
+  await bot.postMessage(channel.id, messageText, { as_user: true });
 }
 
+const postRankingChangeMessage = async (oldRankings, newRankings) => {
+  const rankingChanges = []
+  oldRankings.forEach((oldPlayer, index) => {
+    const newPlayer = newRankings[index]
+    if (newPlayer.id !== oldPlayer.id) {
+      if (index <= 4) {
```

Comment:
pls remove this if(index<=4) as well and put in a separate function. or at the very least use maxSignificantIndex here

Reply:
I've already created a separate function, I just forgot to remove this piece of code